index.html

```diff
@@ -0,0 +1,347 @@
+                work and experience! If you like what you see and have a project you need coded, don’t hesitate to
+                contact me.
+            </p>
+        </div>
+
+        <div class="showcase-contact">
+            <a href="" class="contact-link">LET'S CONNECT</a>
+
+            <div class="showcase-contact-icons">
+                <a href="">
+                    <svg class="showcase-icon" width="20" height="20" viewBox="0 0 20 20" fill="currentColor"
+                        xmlns="http://www.w3.org/2000/svg">
+                        <path
+                            d="M19.7587 0.394423C19.4253 -0.00424693 18.8587 -0.103914 18.4253 0.195088L16.492 1.52399C15.592 0.560536 14.3253 0.0289756 12.9587 0.0289756C10.2587 0.0289756 8.02532 2.22166 8.02532 4.94591C8.02532 5.34458 8.05865 5.71002 8.15865 6.07547C5.82532 5.57713 4.42532 3.88279 3.89198 0.826316C3.82532 0.427646 3.52532 0.128643 3.12532 0.0289756C2.75865 -0.070692 2.35865 0.0954206 2.12532 0.394423C2.05865 0.460868 -3.44135 8.10205 4.52532 14.5804C3.75865 16.2083 1.45865 17.6037 0.558651 18.0688C0.125318 18.2681 -0.0746825 18.7332 0.0253175 19.1984C0.125318 19.6635 0.525318 19.9625 0.991984 19.9625H2.72532C12.8587 19.9625 17.4587 14.9459 17.5587 3.71667L19.6587 1.72332C20.0587 1.3911 20.092 0.793093 19.7587 0.394423Z"
+                            fill="currentColor" />
+                    </svg>
+                </a>
+
+                <a href="">
+                    <svg width="13" height="17" viewBox="0 0 13 17" fill="none" xmlns="http://www.w3.org/2000/svg">
+                        <path
+                            d="M6.64485 0.0345459C3.15607 0.0345459 0.289062 2.86213 0.289062 6.37937V16.3104C0.289062 16.6897 0.599943 17.0001 0.979909 17.0001H4.57231C4.95228 17.0001 5.26316 16.6897 5.26316 16.3104V6.65524C5.26316 5.89661 5.88492 5.27592 6.64485 5.27592C7.40478 5.27592 8.02654 5.89661 8.02654 6.65524V16.3104C8.02654 16.6897 8.33742 17.0001 8.71739 17.0001H12.3098C12.6898 17.0001 13.0006 16.6897 13.0006 16.3104V6.37937C13.0006 2.89662 10.1682 0.0345459 6.64485 0.0345459Z"
+                            fill="currentColor" />
+                    </svg>
+                </a>
+
+                <a href="">
+                    <svg width="20" height="16" viewBox="0 0 20 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+                        <path
+                            d="M18.4042 1.88042L20 0.337684V0H14.4717L10.5317 9.90989L6.04917 0H0.2525V0.337684L2.11667 2.60547C2.29833 2.77305 2.39333 3.01811 2.36917 3.26484V12.1768C2.42667 12.4977 2.32333 12.8278 2.1 13.0611L0 15.6328V15.9663H5.95417V15.6286L3.85417 13.0611C3.62667 12.8269 3.51917 12.5027 3.565 12.1768V4.46821L8.79167 15.9705H9.39917L13.8933 4.46821V13.6312C13.8933 13.8728 13.8933 13.9225 13.7367 14.0808L12.12 15.6615V16H19.9633V15.6623L18.405 14.1204C18.2683 14.016 18.1975 13.8408 18.2267 13.6707V2.33011C18.1975 2.15916 18.2675 1.984 18.4042 1.88042Z"
```

Comment:
- [ ] As required for this project the list of icons uses a `ul` tag to group them and a `li` per icon; each icon has a link (`a` tag), kindly fix this for semantic reasons as well :wink: